src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -195,6 +200,7 @@ private function handleConfig(ContainerBuilder $container, array $config, array
         $container->setAlias('api_platform.path_segment_name_generator', $config['path_segment_name_generator']);
 
         if ($config['name_converter']) {
+            @trigger_error('Configuring "name_converter" within the "api_platform" node is deprecated and will be removed in 3.0. Please use the "serializer.name_converter" node instead.', E_USER_DEPRECATED);
```

Comment:
https://symfony.com/doc/current/components/config/definition.html#deprecating-the-option

Reply:
whoops yes indeed thanks